x-pack/auditbeat/processors/sessionmd/provider/kerneltracingprovider/metrics.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build linux && (amd64 || arm64) && cgo
+
+package kerneltracingprovider
+
+import (
+	"github.com/elastic/elastic-agent-libs/monitoring"
+)
+
+type Stats struct {
+	Insertions      *monitoring.Uint
+	Removals        *monitoring.Uint
+	Aggregations    *monitoring.Uint
+	NonAggregations *monitoring.Uint
+	Lost            *monitoring.Uint
+}
+
+func NewStats(reg *monitoring.Registry) *Stats {
+	return &Stats{
+		Insertions:      monitoring.NewUint(reg, "insertions"),
+		Removals:        monitoring.NewUint(reg, "removals"),
+		Aggregations:    monitoring.NewUint(reg, "aggregations"),
+		NonAggregations: monitoring.NewUint(reg, "nonaggregations"),
+		Lost:            monitoring.NewUint(reg, "lost"),
+	}
+}
```

Comment:
Do we really need a new file for this?

Reply:
It's its own thing with its own object, so I feel like it keeps the code a bit more organized?

Reply:
fine by me